js/misc/paygManager.js

```diff
@@ -350,6 +375,16 @@ var PaygManager = new Lang.Class({
         this._proxy.ClearCodeRemote();
     },
 
+    validateCode(code, partial = false) {
```

Comment:
Nitpick: remove extra spaces around `=`

Reply:
Might be nice to include this in the HACKING style guide.